app/assets/stylesheets/partials/_modal.scss

```diff
@@ -0,0 +1,84 @@
+#modalOverlay {
+  background-color: white;
+  display: none; 
+  height: 100%;
+  left: 0;
+  margin: 0;
+  opacity: 0.7;
+  padding: 0;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 9999;
+}
+
+#modal {
+  background-color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.